libbeat/_meta/config.reference.yml

```diff
@@ -217,6 +217,16 @@
 #    max_depth: 1
 #    target: ""
 #    overwrite_keys: false
+#
+# The following example copies the value of message to message_copied
+#
+#processors:
+#- copy_fields:
+#    fields:
+#        - from: message
+#          to: message_copied
+#    fail_on_error: false
+#    ignore_missing: true
```

Comment:
Seems like this is exactly the opposite to what we have in the defaults in the code for fail_on_error and ignore_missing. Is that on purpose?

Reply:
No. Thanks for finding it.

Reply:
I think this is still missing?